packages/vaadin-combo-box/test/dropdown.test.js

```diff
@@ -7,6 +7,7 @@ describe('dropdown', () => {
 
   beforeEach(() => {
     dropdown = fixtureSync(`<vaadin-combo-box-dropdown></vaadin-combo-box-dropdown>`);
+    dropdown.positionTarget = document.createElement('button');
```

Comment:
Due to the new two-way binding on `opened`, changing `overlay.opened` will propagate back to the dropdown, which then triggers `_openedChanged`. That method requires a `positionTarget` to be set.